package.json

```diff
@@ -52,33 +52,37 @@
     "@typescript-eslint/eslint-plugin": "^6",
     "@typescript-eslint/parser": "^6",
     "aws-cdk": "2.139.0",
-    "esbuild": "^0.20.2",
+    "esbuild": "^0.21.4",
     "eslint": "^8",
     "eslint-import-resolver-typescript": "^3.6.1",
     "eslint-plugin-import": "^2.29.1",
     "jest": "^29.7.0",
     "jest-junit": "^15",
-    "projen": "^0.81.2",
+    "projen": "^0.82.4",
     "standard-version": "^9",
-    "ts-jest": "^29.1.2",
+    "ts-jest": "^29.1.4",
     "ts-node": "^10.9.2",
     "typescript": "^5.4.5"
   },
   "dependencies": {
     "aws-cdk-github-oidc": "^2.4.1",
     "aws-cdk-lib": "2.139.0",
-    "cloudstructs": "^0.9.5",
+    "cloudstructs": "^0.9.8",
     "constructs": "^10.0.5"
   },
   "engines": {
     "node": ">= 20.0.0"
   },
   "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
   "version": "0.0.0",
   "jest": {
+    "coverageProvider": "v8",
```

Comment:
Update the `jest` configuration in `package.json` to match the changes made in the `.projen/tasks.json` file, ensuring consistency across the project. [medium]